src/rp2_common/hardware_irq/include/hardware/irq.h

```diff
@@ -286,6 +286,33 @@ void irq_set_mask_n_enabled(uint n, uint32_t mask, bool enabled);
  * This method will assert if there is already any sort of interrupt handler installed
  * for the specified irq number.
  *
+ * NOTE: By default, the SDK uses a single shared vector table per core, and the current installed
```

Comment:
"single shared... per core" seems a bit contradictory? Would "a single shared vector table for both cores" make more sense?
Also, "current installed" should probably be "currently installed" ?

Reply:
pushed directly to develop